docs/03-policies/security.md

```diff
@@ -181,27 +188,27 @@ Developers and themers working on the site codebase (and committing code to Git)
   - Session hijacking, prevented by using SSL and correct site/session settings.
   - Data disclosure, prevented by carefully setting and testing access control, as well as using SSL as needed.
   - Password guessing attacks, mitigated by using strong passwords.
-- Software that is not licensed under an approved CivicActions open source license may not be used on a project without prior approval from the legal team.
+- Avoid inclusion of any software not licensed under an approved CivicActions open source license on a project without prior approval from the legal team.
 
 The project technical lead (or a designated lead engineer/lead themer or peer-review process) is responsible for reviewing all new/modified code each sprint, and ensuring it meets a high standard of quality.
 
 Developers and themers maintaining local sandbox copies of client sites must also:
 
 - Ensure that our standard tools for creating, sanitizing and transferring database dumps for sandboxes are used.
-- Ensure that unsanitized mysql data (created via mysqldump or phpmyadmin) to sandboxes is not downloaded from the server to a local sandbox.
+- Ensure that unsanitized mysql data (extracted via mysqldump or phpmyadmin) is not downloaded from the server to a local sandbox.
 - Ensure that all confidential data associated with a project (such as databases, database dumps and other files) are securely deleted from their system(s) when leaving or completing a project.
 
 Developers and themers working on the site vhost (SSH/shell, file system, database) must also:
 
-- Ensure they follow best practices with respect to SSH keys, passphrases and passphrase caching (see above).
-- SSH, SFTP and SCP access to vhosts are managed exclusively by designated admin(s). Access by password, manually installed SSH keys (other than by admins), web based "shell" script, port forwarding to 3rd parties or other methods are forbidden by unless authorized in advance by the CIO.
-- Temporary SSH port forwarding is permitted for the purpose of accessing the server MySQL from your own desktop.
-- Accessing CivicActions servers by initiating an SSH connection from external client or 3rd party servers is strongly discouraged - it is preferable to SSH out from the CivicActions server.
-- Running non-standard software on a vhost requires a system ticket and approval from a member of the IT team. This includes:
+- Follow best practices with respect to SSH keys, passphrases and passphrase caching (see above).
+- Access the server only by routes (e.g. SSH, SFTP, SCP) configured by designated admins. Access by password, manually installed SSH keys (other than by admins), web based "shell" script, port forwarding to 3rd parties or other methods are forbidden, unless authorized in advance by the CIO.
```

Comment:
We don't have a CIO. We have a CTO. 

Instead of "routes", I think a better word would be "methods"